fun main() {
    // Write some code to cause an exception
    val number: Int = 5
    number / 0
}
